source/vibe/http/websockets.d

```diff
@@ -64,55 +64,60 @@ class WebSocketException: Exception
 }
 
 /**
-	Returns a HTTP request handler that establishes web socket conections.
+    Establishes a web socket conection and passes it to the $(D on_handshake) delegate.
 */
-HTTPServerRequestDelegate handleWebSockets(WebSocketHandshakeDelegate on_handshake)
+void doHandleWebsocket(WebSocketHandshakeDelegate on_handshake, HTTPServerRequest req, HTTPServerResponse res)
```

Comment:
The `on_handshake` parameter could be `scope WebSocketHandshakeDelegate` here to avoid creating heap delegates at the call site.